pkgs/top-level/all-packages.nix

```diff
@@ -1584,6 +1584,14 @@ with pkgs;
 
   brewtarget = libsForQt514.callPackage ../applications/misc/brewtarget { } ;
 
+  stdenvBootstrapTools =
+    let args = { crossSystem = stdenv.hostPlatform.system; }; in
```

Comment:
Github seems to have garbage-collected my commit after I squashed the PR.

The addition of this line fixes the ["embarrassing oversight"](https://github.com/NixOS/nixpkgs/pull/168650#issuecomment-1109381005) I referred to.